azureadps-1.0/MSOnline/Get-MsolDevice.md

```diff
@@ -39,6 +44,10 @@ Get-MsolDevice -ObjectId <Guid> [<CommonParameters>]
 Get-MsolDevice -RegisteredOwnerUpn <String> [<CommonParameters>]
 ```
 
+### GetDeviceIncludeSystemManaaged
+```
+Get-MsolDevice -IncludeSystemManagedDevices [<CommonParameters>]
```

Comment:
You need to include the -All flag here:
`Get-MsolDevice -All -IncludeSystemManagedDevices [<CommonParameters>]`